src/main/java/com/adyen/model/checkout/CreatePaymentLinkRequest.java

```diff
@@ -59,24 +79,99 @@
+     */
+    @JsonAdapter(RecurringProcessingModelEnum.Adapter.class)
+    public enum RecurringProcessingModelEnum {
+        CARDONFILE("CardOnFile"),
+        SUBSCRIPTION("Subscription"),
+        UNSCHEDULEDCARDONFILE("UnscheduledCardOnFile");
+
+        @JsonValue
+        private String value;
+
+        RecurringProcessingModelEnum(String value) {
+            this.value = value;
+        }
+
+        public String getValue() {
+            return value;
+        }
+
+        @Override
+        public String toString() {
+            return String.valueOf(value);
+        }
+
+        public static RecurringProcessingModelEnum fromValue(String text) {
+            for (RecurringProcessingModelEnum b : RecurringProcessingModelEnum.values()) {
+                if (String.valueOf(b.value).equals(text)) {
+                    return b;
+                }
+            }
+            return null;
+        }
+
+        public static class Adapter extends TypeAdapter<RecurringProcessingModelEnum> {
+            @Override
+            public void write(final JsonWriter jsonWriter, final RecurringProcessingModelEnum enumeration) throws IOException {
+                jsonWriter.value(enumeration.getValue());
+            }
+
+            @Override
+            public RecurringProcessingModelEnum read(final JsonReader jsonReader) throws IOException {
+                String value = jsonReader.nextString();
+                return RecurringProcessingModelEnum.fromValue(String.valueOf(value));
+            }
+        }
+    }
+
+    @SerializedName("recurringProcessingModel")
+    private RecurringProcessingModelEnum recurringProcessingModel = null;
+
     @SerializedName("reference")
     private String reference = null;
 
     @SerializedName("returnUrl")
     private String returnUrl = null;
 
+    @SerializedName("reusable")
+    private Boolean reusable = null;
+
+    @SerializedName("riskData")
+    private RiskData riskData = null;
+
     @SerializedName("shopperEmail")
     private String shopperEmail = null;
 
     @SerializedName("shopperLocale")
     private String shopperLocale = null;
 
+    @SerializedName("shopperName")
+    private Name shopperName = null;
+
     @SerializedName("shopperReference")
     private String shopperReference = null;
 
+    @SerializedName("splits")
+    private List<Split> splits = null;
```

Comment:
can you please update `Split.TypeEnum`? The possible value "BalanceAccount" has been added to the specs.

Reply:
updated all dependent models too.